src/fixate/drivers/dmm/fluke_8846a.py

```diff
@@ -341,3 +342,11 @@ def analog_filter(self, bandwidth=None):
         elif any(x in self._mode for x in ["dc", "res"]):
             self._write(self._filters[self.mode] + ":FILT:STAT ON")
         pass
+
+    def get_serial(self):
+        """
+        serial string returned by *IDN?
+        outg_sw_date is the
+        :return: FLUKE,8846A,S/N
+        """
+        return self.instrument.query("*IDN?")
```

Comment:
Does query return bytes or string? I'm inclined to think we should be explicit here that get_serial (or get_identity if we implement that) should return a string and not a bytes object, otherwise we potentially need to know about the encoding used elsewhere in the code base.

Reply:
Returns a string